docs/content/docs/dev/datastream/operators/windows.md

```diff
@@ -814,6 +926,27 @@ input
 
 ```
 {{< /tab >}}
+{{< tab "Python" >}}
+```python
+input = ...  # type: DataStream
+
+input \
+  .key_by(<key selector>) \
+  .window(<window assigner>) \
+  .reduce(lambda v1, v2: (v1[0], v1[1] + v2[1]),
```

Comment:
The implementation is incorrect. It doesn't match the Java/Scala example, also doesn't match the description of this section: "The following example shows how an incremental ReduceFunction can be combined with a ProcessWindowFunction to return the smallest event in a window along with the start time of the window."